libcaf_core/caf/actor_storage.hpp

```diff
@@ -132,7 +132,7 @@ void intrusive_ptr_release(actor_storage<T>* x) {
   // and destroy the data block
   if (x->ctrl.strong_refs.fetch_sub(1, std::memory_order_acq_rel) == 1) {
     x->destroy_data();
-    intrusive_ptr_relase_weak(x);
+    intrusive_ptr_release_weak(x);
```

Comment:
This is the only place where we've misspelled `intrusive_ptr_release_weak`, which means this should've resulted in a compiler error. But it didn't. Dead code?

Seems like it when looking at the [coverage report](https://jenkins.inet.haw-hamburg.de/job/CAF/job/actor-framework/job/master/378/cobertura/caf_test/libcaf_core_caf_actor_storage_hpp/). Can you check whether we can just remove all the `intrusive_ptr_` functions from `actor_storage.hpp`? As far as I can tell, we never form an `intrusive_ptr` to an actor storage. Only to an actor control block.

Reply:
I will check that tomorrow.

Reply:
Done.